### ISSUE TITLE
Views aren't excluded from schemawatch, leading to nuisance staging-table-not-found messages

### ISSUE THREAD

**Issue by sjuls** [cockroachdb/cdc-sink#617]
Problem;
We have database in cockroach we would like to migrate to a new cluster, however this database contains a view in its schema leading to errors in cdc-sink when the view is present in the target database. This also results in data from the changefeed not being applied to the target tables.

Error message
```
"relation \"movr.cdcsink.movr_public_sneaky_view\" does not exist", \"file\": \"errors.go\",\n \"line\": 171,\n \"routine\": \"NewUndefinedRelationError\"
```

Script to reproduce: [demo_up.sh](https://github.com/sjuls/cdc-sink/blob/view-erroring/scripts/shell_demo/demo_up.sh)

**Comment by bobvawter** [comment]
It looks like views are not being excluded from the `SchemaData.Order` mapping, which is what drives the resolver.

**Comment by sjuls** [comment]
> In `demo-up.sh`, permissions on the `leases` tables are being reassigned to `root`; that yields the following error: `user cdcsink does not have UPDATE privilege on relation leases`. This is unrelated to the views not being excluded from the schema data.

Yeah, that was testing a different failure mode, forgot to clean that up.

**Comment by sjuls** [comment]
I see you changed the title, just want to confirm that it is not just a nuisance, but actually blocking data from being synced? Even removing the leases grant fiddling I still see data missing after the workload completes and cdc-sink has time to flush.

**Comment by bobvawter** [comment]
It should have been fixed in PR #618.  Did that not work for you?

**Comment by sjuls** [comment]
Hi @bobvawter ,

Yeah, it is fixed after your merge, thanks! Just wanted to make sure I understand the root of the missing data.
